src/server.tsx

```diff
@@ -107,19 +95,17 @@ const buildEpic = async (
         countryCode: targeting.countryCode,
     };
 
-    return asResponse(getEpic(props), testTracking, result.debug);
+    return { variant, meta: testTracking, debug: result.debug };
```

Comment:
Should we include some identifier for the Epic component here, that the platform could use to dynamically build the URL it needs to fetch the module from? Something like `module: 'epic.js'`

Reply:
Really nice idea.